guides/common/modules/proc_installing-insights-iop-using-import-and-export.adoc

```diff
@@ -18,25 +18,75 @@ ifndef::satellite[]
 # {project-package-install} skopeo
 ----
 endif::[]
-* The disconnected {ProjectServer} is prepared to receive the image.
+* The disconnected {ProjectServer} is prepared to receive the images.
 
 .Procedure
-. On the connected system, export the container image:
+. On the connected system, export the container images:
```

Comment:
Perhaps state explicitly that it's now a script? You could write something like "Save the following as a script and run it on the connected system."